README.md

```diff
@@ -1,19 +1,218 @@
+  policy, and the text describes the details of the policy violation. If the
+  issue is already open, it is pinged with a comment every 24 hours (not
+  currently user configurable). Once the violation is addressed, the issue
+  should be closed by Allstar within 5-10 minutes.
+- `fix`: This action is policy specific. The policy will make the changes to the
+  GitHub settings to correct the policy violation. Not all policies will be able
+  to support this (see below).
+
+Proposed, not yet implemented, actions. Further discussion and definition needed.
+
+- `block`: Allstar can set a [GitHub Status
+  Check](https://docs.github.com/en/github/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/about-status-checks)
+  and block any PR in the repository from being merged.
+- `email`: Allstar could send an email somehow.
+- `rpc`: Allstar could send an rpc to some organization-specific system.
+
+## **Policies**
+
+Similar to the Allstar app enable configuration, all policies are enabled and
+configured with a yaml file in either the organization's `.allstar` repository,
+or the repository's `.allstar` directory. As with the app, policies are opt-in
+by default, also the default `log` action won't produce visible results. A
+simple way to enable all policies is to create a yaml file for each policy with
+the contents:
+
+```
+optConfig:
+  optOutStrategy: true
+action: issue
+```
+
+
+### Branch Protection
+
+This policy's config file is named `branch_protection.yaml`, and the [config
+definitions are
+here](https://pkg.go.dev/github.com/ossf/allstar@v0.0.0-20210728182754-005854d69ba7/pkg/policies/branch#OrgConfig).
+
+The branch protection policy checks that GitHub's [branch protection
+settings](https://docs.github.com/en/github/administering-a-repository/defining-the-mergeability-of-pull-requests/about-protected-branches)
+are setup correctly according to the specified configuration. The `fix` action
+is not yet implemented, but should be possible and is planned.
+
+### Binary Artifacts
+
+This policy's config file is named `binary_artifacts.yaml`, and the [config
+definitions are
+here](https://pkg.go.dev/github.com/ossf/allstar@v0.0.0-20210728182754-005854d69ba7/pkg/policies/binary#OrgConfig).
+
+This policy incorporates the [check from scorecard](https://github.com/ossf/scorecard/#scorecard-checks). The `fix` action is not planned.
+
+### Outside Collaborators
+
+This policy's config file is named `outside.yaml`, and the [config definitions
+are
+here](https://pkg.go.dev/github.com/ossf/allstar@v0.0.0-20210728182754-005854d69ba7/pkg/policies/outside#OrgConfig).
+
+This policy checks if any [Outside
+Collaborators](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/adding-outside-collaborators-to-repositories-in-your-organization)
+have either administrator(default) or push(optional) access to the
+repository. Typically only organization members have this access.
```

Comment:
s/Typically only organization members have this access./Only organization member should have this access, as otherwise untrusted members can change admin level settings and commit malicious code.

Reply:
done